### PR TITLE
Add tests for DashboardController

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

**Review by Ullmie02 — APPROVED** [review]

**Review by barronpm — APPROVED** [review]

File: tests/Jellyfin.Api.Tests/Controllers/DashboardControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+        [Fact]
+        public async Task GetDashboardConfigurationPage_ExistingPage_CorrectPage()
+        {
+            var client = _factory.CreateClient();
+
+            var response = await client.GetAsync("/web/ConfigurationPage?name=TestPlugin").ConfigureAwait(false);
+
+            Assert.True(response.IsSuccessStatusCode);
+            Assert.Equal("text/html", response.Content.Headers.ContentType?.MediaType);
```
Comment (crobibero) [inline tests/Jellyfin.Api.Tests/Controllers/DashboardControllerTests.cs:39]:
```suggestion
            Assert.Equal(MediaTypeNames.Text.Html, response.Content.Headers.ContentType?.MediaType);
```

File: tests/Jellyfin.Api.Tests/Controllers/BrandingControllerTests.cs [hunk 2]
```diff
@@ -24,8 +24,9 @@ public async Task GetConfiguration_ReturnsCorrectResponse()
             var response = await client.GetAsync("/Branding/Configuration");
 
             // Assert
-            response.EnsureSuccessStatusCode();
-            Assert.Equal("application/json; charset=utf-8", response.Content.Headers.ContentType?.ToString());
+            Assert.True(response.IsSuccessStatusCode);
+            Assert.Equal("application/json", response.Content.Headers.ContentType?.MediaType);
```
Comment (crobibero) [inline tests/Jellyfin.Api.Tests/Controllers/BrandingControllerTests.cs:28]:
```suggestion
            Assert.Equal(MediaTypeNames.Application.Json, response.Content.Headers.ContentType?.MediaType);
```

File: tests/Jellyfin.Api.Tests/Controllers/BrandingControllerTests.cs [hunk 3]
```diff
@@ -24,8 +24,9 @@ public async Task GetConfiguration_ReturnsCorrectResponse()
...
+            Assert.Equal("utf-8", response.Content.Headers.ContentType?.CharSet);
```
Comment (crobibero) [inline tests/Jellyfin.Api.Tests/Controllers/BrandingControllerTests.cs:29]:
```suggestion
            Assert.Equal(Encoding.UTF8.BodyName, response.Content.Headers.ContentType?.CharSet);
```

File: tests/Jellyfin.Api.Tests/Controllers/BrandingControllerTests.cs [hunk 4]
```diff
@@ -42,7 +43,7 @@ public async Task GetCss_ReturnsCorrectResponse(string url)
             var response = await client.GetAsync(url);
 
             // Assert
-            response.EnsureSuccessStatusCode();
+            Assert.True(response.IsSuccessStatusCode);
             Assert.Equal("text/css; charset=utf-8", response.Content.Headers.ContentType?.ToString());
```
Comment (crobibero) [inline tests/Jellyfin.Api.Tests/Controllers/BrandingControllerTests.cs:47]: Should this also be split?

**Review by crobibero — APPROVED** [review]

**Pull request merged**